virttest/utils_hotplug.py

```diff
@@ -493,7 +493,8 @@ def vcpuhotunplug_unsupport_str():
 
 
 def create_mem_xml(tg_size, pg_size=None, mem_addr=None, tg_sizeunit="KiB",
-                   pg_unit="KiB", tg_node=0, node_mask=0, mem_model="dimm"):
+                   pg_unit="KiB", tg_node=0, node_mask=0, mem_model="dimm",
+                   lb_size=None, lb_sizeunit="Kib", mem_access=None):
```

Comment:
Please also add description for adding params

Reply:
Updated